lib/utils.ts

```diff
@@ -0,0 +1,17 @@
+import { SALEOR_DOMAIN_HEADER } from "@saleor/app-sdk/const";
```

Comment:
I was out of ideas on how this file should be named. Any suggestions? Would like to avoid "utils" kitchensinks.